### PR TITLE
`disabled` option for <TextArea>

### PR DESCRIPTION
**Pull request by axross**

### CODE REVIEW

**Comment by axross** [comment]
@ryotogashi Could you have a review when you have hands free? Thanks!

File: components/TextArea/TextArea.tsx [hunk 1]
```diff
@@ -65,15 +73,24 @@ const Root = styled.textarea<{
   resize: ${({ resizable }) => (resizable ? "vertical" : "none")};
   transition: border 200ms ease, transform 200ms ease, box-shadow 200ms ease;
 
-  &:hover,
-  &:focus {
+  ::placeholder {
```
Comment (ryotogashi) [inline components/TextArea/TextArea.tsx]: @axross Why did you delete & ?

Comment (axross) [reply components/TextArea/TextArea.tsx]: `&` refers itself. And referring itself is automatically completed by nesting. You need it only when using `& > .something` or `&.something`.

**Comment by axross** [comment]
Thanks!
